### DISCUSSION TITLE
Crashing when trying to get level above 50

### DISCUSSION THREAD

**Discussion by ChestWithCrown** [R2Northstar/Northstar#428] [Tech Support] [answered]
By using `script GetPlayerArray()[0].SetPersistentVar("xp", 9999)` I managed to set my level above 50 and now every time the game tries to render my level it crashes to main menu with `sh_xp.gnu R2Northstar/NorthstarLauncher#162 [UI] Index "51" is beyond of array size of 51`.

**Comment by GeckoEidechse** [comment, marked as answer]
You might be able to change the value from within main menu but not sure tbh.

I'd just recommend resetting your levels and loadouts as described here: https://r2northstar.gitbook.io/r2northstar-wiki/installing-northstar/troubleshooting#i-used-a-command-to-set-my-player-gun-xp-level-and-i-set-it-too-high-so-now-my-game-crashes-when-try
